### ISSUE TITLE
Unsure if reading correct documentation

### ISSUE THREAD

**Issue by Mandated** [MicrosoftDocs/feedback#1123]
**🛑 IMPORTANT**: You can get your feedback addressed faster if you **use the comment section for the article in which you encountered a problem**.

**Link to article:**
https://docs.microsoft.com/en-us/windows/desktop/WinProg/using-the-windows-headers

**Problem:**
Nothing is really wrong, and I apologize if this is not the correct forum for asking this question.  I have been learning C# for a while now and am branching out to learn .NET and other related documentation - I am not certain sometimes if what I am reading is geared for using C#.

It seems like C++ is the prerequisite language for many of the articles I've found when looking through Desktop programming, and just wanted clarification on whether C# can interop with the sections I'm looking at before I invest a lot of time into reading them.

Basically, anything under Window Headers is what I'm referencing. 

Also, I have spent a while now reading up on XAML in both WPF & in UWP although UWP is what I am planning to try my hand at.  Is UWP a good place to start for developing apps with C#?
